code/__defines/atmos.dm

```diff
@@ -50,8 +50,8 @@
 #define CARBON_LIFEFORM_FIRE_DAMAGE     4
 
 // Phoron fire properties.
```

Comment:
[squints]

Reply:
Will get these in a future PR, as discussed.